clang/test/OpenMP/amdgpu_threadprivate.cpp

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: asserts
```

Comment:
This does not require asserts

Reply:
The test crashes with an error only with an `asserts` build.
Should I just that aspect in a comment, and remove the `asserts` requirement?

Reply:
The test should not crash under any build. It does not matter what the symptoms are today